### DISCUSSION TITLE
GitHub aprrove

### DISCUSSION THREAD

**Discussion by vixonblues23** [community/community#52930] [GitHub Education]
How do I get this GitHub student thing approved? This has been a big pain in the butt for the last couple of weeks now. Everything I have sent has been denied.

**Comment by vixonblues23** [comment]
You know what I sent proof that I am currently taking informatics 109 this semester spring 2023. I sent my student ID, one of my transcripts it asked for, the syllabus for the class currently. I sent everything that I can think of to answer those stupid requests your system asks for, and yet it’s not good enough. I am to the point I don’t really care if your stupid system denies me or not. All I have to say this is the stupidest and worst program I have ever dealt with to get approved, it shouldn’t be this difficult to get approved. I am sick of trying over and over and not getting anywhere. I wish this program never existed, it is too much of a hassle to deal with.
________________________________
From: Mntwana ***@***.***>
Sent: Sunday, April 16, 2023 7:40 AM
To: community/community ***@***.***>
Cc: Kristen Antala ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL]Re: [community/community] GitHub aprrove (Discussion #52930)


Hi @vixonblues23<https://github.com/vixonblues23>

The trick is to make sure you follow the instructions very carefully. All you need is a school-issued email address and/or official, dated proof of enrollment

If your documents keep being rejected you might want to try using different documents to prove you enrollment. But for more specific assistance it would be easier if you provided reasons for you application being rejected.

—
Reply to this email directly, view it on GitHub<https://github.com/orgs/community/discussions/52930#discussioncomment-5628258>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A7CG55F35FAV3TRFFTDUVXTXBPSFJANCNFSM6AAAAAAW7TN4IY>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

STUDENT EMAIL NOTICE: This communication originated from an Ivy Tech Community College student email address. Any correspondence from this email address is not official Ivy Tech business communications or interests and does not represent the opinions or positions of Ivy Tech Community College.

**Reply by samus-aran** [reply]
> I wish this program never existed...

Ack, we're sorry to hear that @vixonblues23. We provide this program to _millions_ of students every year and yes we do have some that have difficulties getting verified but I assure you this is in the minority. Understandably you're frustrated but usually it's just a matter of guidance.

Let's see if we can assist ⭐ 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. 

You say you have used a Student ID before, but did your profile details correspond to this piece of evidence. Usually combining that with an academic email is the best way to being accepted unless your profile details were not up to date at the time of submission. Just double check your Student ID + profile info is correct, log out and then log back in again before re-applying. 

Let us know how you get on 👍

**Comment by vixonblues23** [comment]
Oh and I got a bad grade for this stupid GibHub account that I setup because your dumb system kept denying me.
________________________________

**Comment by vixonblues23** [comment]
I am done with your dumb program, I hope I never have to use this for anything tech related in the future.
________________________________
